rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.put_mapping/10_basic.yml

```diff
@@ -145,28 +145,6 @@
   - is_false: test_index.mappings.properties.foo.meta.bar
   - match: { test_index.mappings.properties.foo.meta.baz: "quux" }
 
----
-"disabling synthetic source fails":
```

Comment:
maybe replace with a test that tries to update the new index setting?

Reply:
This is `indices.put_mapping` test :) we have this elsewhere..